OpenRA.Game/Game.cs

```diff
@@ -800,7 +802,9 @@ static void Loop()
 					var haveSomeTimeUntilNextLogic = now < nextLogic;
 					var isTimeToRender = now >= nextRender;
 
-					if ((isTimeToRender && haveSomeTimeUntilNextLogic) || forceRender)
+					// Disable all rendering while the window is suspended
+					// Works around the threaded renderer bug on Windows
```

Comment:
can you add link to this bug for documentation reference?

Reply:
Good idea actually.